src/mesh.jl

```diff
@@ -173,21 +173,23 @@ function XYZ end
 
 Base.size(g::Grid) = size(topology(g))
 
-Base.minimum(g::Grid{Dim}) where {Dim} = cart2vert(g, ntuple(i -> 1, Dim))
-Base.maximum(g::Grid{Dim}) where {Dim} = cart2vert(g, size(g) .+ 1)
+Base.minimum(g::Grid{Dim}) where {Dim} = vertex(g, ntuple(i -> 1, Dim))
+Base.maximum(g::Grid{Dim}) where {Dim} = vertex(g, size(g) .+ 1)
 Base.extrema(g::Grid{Dim}) where {Dim} = minimum(g), maximum(g)
 
+vertex(grid::Grid, ijk::Tuple) = vertex(grid,  LinearIndices(size(grid) .+ 1)[ijk...])
```

Comment:
Maybe we can use `Dims{Dim}` to be safer?
```suggestion
vertex(grid::Grid{Dim}, ijk::Dims{Dim}) where {Dim} = vertex(grid,  LinearIndices(size(grid) .+ 1)[ijk...])
```

Reply:
Agree, and this fallback should show up right after the first appearance of the function